wp-content/themes/core/components/image/css/image.pcss

```diff
@@ -41,16 +33,35 @@
 		width: 100%;
 	}
 
-	/* CASE: Apply an overlay. */
-	.c-image--overlay & {
+	/* CASE: Using image in figure tag, (not as a background). */
+	&.c-image--figure {
```

Comment:
I'd probably turn this into more of a utility sort class, something like `c-image--object-fit`

Reply:
+1